pom.xml

```diff
@@ -293,6 +293,12 @@
                     <configuration>
                         <argLine>${argLine} -Duser.timezone=UTC -Dddf.version=${ddf.version}
                         </argLine>
+                        <includes>
+                            <include>**/*Test.java</include>
+                            <include>**/Test*.java</include>
+                            <include>**/*TestCase.java</include>
+                            <include>**/*Spec.class</include>
```

Comment:
❗️ If we don't have test classes that match the `**/*Test*.java` and `**/*TestCase.java` patterns, we should remove those and only keep the ones that match our guidelines, i.e., `**/*Test.java` and `**/*Spec.class`.